pkg/shell/indexes.js

```diff
@@ -143,6 +143,18 @@ function MachinesIndex(index_options, machines, loader, mdialogs) {
     });
 
     function show_disconnected() {
+        if (!ready) {
+            let ca_cert_url = window.sessionStorage.getItem("CACertUrl");
```

Comment:
This isn't being set anywhere, and I don't think it's a standard attribute of sessionStorage. So it looks like the `#safari` part never works?

Reply:
Nevermind, it's set in the previous commit.